projects/assets-library/src/icons/icon-type.ts

```diff
@@ -78,6 +79,7 @@ export const enum IconType {
   NoData = 'svg:noData',
   Node = 'node',
   None = 'none',
+  Note = 'svg:note',
```

Comment:
both svgs `page` and `note` looks to be the same. If it already exists, could we reuse that and avoiding a new `note` icon?

Reply:
They are same, I liked `note` better name for that icon and more generic as it is used as `description` icon in the security tab inside settings.
I will be changing that on the traceable side once this is merged.